spec/requests/update_collection.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Why is this file removed? Should this be a different PR?

Reply:
This has been removed in master so shouldn't need to be in this branch.

Reply:
@justinlittman please rebase then